test/performance/range/views/view_filter_benchmark.cpp

```diff
@@ -0,0 +1,96 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <vector>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alphabet/all.hpp>
+#include <seqan3/std/ranges>
+#include <seqan3/test/performance/sequence_generator.hpp>
+
+/** This benchmarks evaluates views::filter against other forms of filtering **/
+
+auto const happy  = [] (seqan3::dna4 const c) { return c.to_rank() < 42; }; // always true
```

Comment:
```suggestion
auto const always_true  = [] (seqan3::dna4 const c) { return c.to_rank() < 42; }; // always true
```

Reply:
"Happy" vs "sad" is a well-established term:
https://en.wikipedia.org/wiki/Happy_path

But I have no strong feelings about this.

Reply:
never heard of it.